net/dns/manager.go

```diff
@@ -52,22 +52,27 @@ type Manager struct {
 
 	resolver *resolver.Resolver
 	os       OSConfigurator
+	goos     string // empty defaults to linux
```

Comment:
this should actually imply `runtime.GOOS` no?

Reply:
done